config/config.go

```diff
@@ -324,7 +324,6 @@ type Config struct {
 	HttpServerOptions         HttpServerOptionsConfig `json:"http_server_options"`
 	ReloadWaitTime            int                     `bson:"reload_wait_time" json:"reload_wait_time"`
 	VersionHeader             string                  `json:"version_header"`
-	UseAsyncSessionWrite      bool                    `json:"optimisations_use_async_session_write"`
```

Comment:
We also should remove `optimisations_use_async_session_write ` from
- `tyk.conf.example`
- `cli/linter/schema.json`
- `install/data/tyk.with_dash.conf`
- `install/data/tyk.self_contained.conf`